### DISCUSSION TITLE
Associazionie in HA dell'interruttore connesso vimar

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#940] [HUB: Home Assistant]
Ciao ragazzi, sapete per caso se l'interruttore connesso della vimar 14592 è integrabile in HA? E' sia Zigbee che bluetooth ma io li vorrei attuare con zigbee.
Grazie.

**Comment by snakuzzo** [comment]
Puoi integrarlo sicuramente tramite ZHA o Zigbee2mqtt. Guarda [QUI](https://zigbee.blakadder.com/Vimar_14592_16492_19592_20592.html)

**Reply by calcioscacchi** [reply]
Grazie! Devo studiarmi tutto, in quanto entrerei in un nuovo mondo. Ho trovato nel sito guide dedicate a Zigbee2mqtt, ma nulla riguardo ZHA. Mi interesserebbe capire come funzionano entrambi per poi capire quale sia meglio per le mie esigenze. 
Abbiamo della documentazione su ZHA? 

Detto questo, mi piacerebbe provare ad aggirare l'integrazione diretta, visto che la mia echo vede l'interruttore, voglio provare a inserire in HA un interruttore booleano e creare una routine in alexa che se il booleano è on = accendi interrutore, se off = spegni interruttore.
Per prova ;)

**Reply by snakuzzo** [reply]
"abbiamo" chi ? :)
Noi non abbiamo ad oggi guide su ZHA.
Ad ogni modo...

- ZHA ti permette di collegarti direttamente ad un coordinator Zigbee via USB o TCP ed è gestito dai dev di HA
- Zigbee2mqtt è un software a parte (che trovi anche come addon se usi HassOS) che si appoggia ad un broker MQTT che a sua volta sarà integrato su HA. E' gestito e manutenuto da Koenkk

A te la scelta

Non ho capito che intendi con "aggirare l'integrazione diretta". Di quale integrazione parli ? Vuoi lasciare il device associato ad Alexa ed esporre semplicemente un "interruttore" su HA da esporre su Alexa ?
Per quanto mi riguarda la trovo una cosa senza alcuna logica, ma magari hai delle esigenze specifiche che mi sfuggono

**Reply by calcioscacchi** [reply]
il senso è riuscire a gestire indirettamente l'interruttore senza dover usare zigbee2mqtt. MA si tratta più che altro di un esperimento, mi documenterò su zigbee2mqtt.
Grazie.
